lib/server/Server.js

```diff
@@ -19,6 +20,7 @@ var Server = {
         var onRouteHandled = config.onRouteHandled;
         var ServerAppToUse = config.ServerApp || ServerApp;
         var appRoot = environmentConfig.appRoot;
+        var proxyClientRequestsToApi = config.proxyClientRequestsToApi;
```

Comment:
In my opinion it should be, but it would not be backwards compatible. I was thinking it could swap for a 1.0 release.